packages/dds/task-manager/README.md

```diff
@@ -19,6 +19,15 @@ To get started, install the package by running the following command:
 npm i @fluidframework/task-manager
 ```
 
+## Importing from this package
+
+This package leverages [package.json exports](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-beta/#packagejson-exports-imports-and-self-referencing) to separate its APIs by support level.
```

Comment:
I'd also link to the node docs to make it doubly clear that this is a standard node thing, not some weird TS-only thing.

Reply:
Yeah, that's fair. I replaced the existing link with a link to Node's docs on it.